docs/stencil-docs/developing-further/customizing-invoices.md

```diff
@@ -73,6 +80,16 @@ This invoice becomes available in a shopper’s orders after completing an order
 
 As a developer, you can customize this invoice by editing your theme’s <span class="fn">invoice.html</span> file that is present in your theme’s files. In Cornerstone, this file is located in <span class="fp">/cornerstone/templates/pages/account/orders/invoice.html</span>.
 
+## Customizing the Detailed Customer Printable Invoice
+
+This invoice is used when a store recieves an order from a United Kingdom address. This invoice can be customized by performing the following steps:
+
+1. Download the [PrintableDetailedInvoice.html](https://storage.googleapis.com/bigcommerce-production-dev-center/template-files/PrintableDetailedInvoice.html) file, which contains the skeleton of the detailed printable invoice.
+
+2. Open the file in the text editor of your choice and edit the HTML file to match your desired customization.
+
+3. Use WebDAV to upload your customized HTML file to WebDAV’s <span class="fp">/template/ folder</span>. We recommend the freeware [WebDAV](https://support.bigcommerce.com/s/article/File-Access-WebDAV#webdav-client) client CyberDuck (Windows and Mac OS), for which your store’s control panel offers a downloadable connection file prefilled with most store credentials.

```

Comment:
**Suggestion:** Use WebDAV to upload your customized HTML file to WebDAV’s <span class="fp">/template/ folder</span>. We recommend **CyberDuck**, the freeware [WebDAV](https://support.bigcommerce.com/s/article/File-Access-WebDAV#webdav-client) client, available for Windows and Mac OS. **Your** store’s control panel offers a downloadable connection file prefilled with most store credentials.